pydantic/main.py

```diff
@@ -84,6 +84,9 @@ class BaseModel(metaclass=_model_construction.ModelMetaclass):
         __pydantic_parent_namespace__: Parent namespace of the model.
         __pydantic_custom_init__: Custom init of the model.
         __pydantic_post_init__: Post init of the model.
+        model_config: Configurations of model.
```

Comment:
```suggestion
        model_config: Configuration settings for the model.
```
Any chance we could move this to the top or bottom of the list of `__pydantic_*__` things? Since the name is a different format, I feel it would be nice if it weren't just in the middle like this